media/css/_navigation.scss

```diff
@@ -0,0 +1,127 @@
+.navbar-inner {
+  background: $sidebar-background;
+  border-width: 0 !important;
+}
+
+.navbar-inverse {
+  .navbar-inner {
+    background: $sidebar-background;
+  }
+
+  .brand:hover,
+  .nav > .active > a,
+  .nav > .active > a:hover,
+  .nav > .active > a:focus {
+    background: $sidebar-link-highlight;
+  }
+}
+
+.navbar {
+  .brand {
+    max-height: 50px;
+    overflow: visible;
+    padding: 0 3px;
+    margin-right: 3px;
+    margin-left: 3px;
+  }
+
+  .nav {
+    font-size: 14px;
+    text-align: left;
+    display: inline;
+    margin: 0;
+    list-style: none;
+
+    li {
+      &:hover {
+        background: $sidebar-link-highlight;
+        color: $white;
+      }
+    }
+
+    > li {
+      > ul {
+        padding: 0;
+        position: absolute;
+        top: 50px;
+        left: 0;
+        min-width: 150px;
+        display: none;
+        opacity: 0;
+        visibility: hidden;
+        -webkit-transiton: opacity 0.2s;
+        -moz-transition: opacity 0.2s;
+        -ms-transition: opacity 0.2s;
+        -o-transition: opacity 0.2s;
+        transition: opacity 0.2s;
+        -webkit-box-shadow: 1px 6px 8px rgba(0,0,0,.175);
+        box-shadow: 1px 6px 8px rgba(0,0,0,.175)
+      }
+
+      &:hover {
+        ul {
+          display: block;
+          opacity: 1;
+          visibility: visible;
+        }
+      }
+
+      > span {
+        display: inline-block;
+        height: 20px;
+        padding: 15px;
+        cursor: pointer;
+      }
+    }
+
+    ul {
+      > li {
+        background: $white;
+        display: block;
+
+        > a {
+          padding: 6px 10px;
+        }
+      }
+    }
+
+    b.caret {
+      border-top-color: $white;
+      border-bottom-color: $white;
+      margin-top: 8px;
+    }
+  }
+
+  ul {
+    @extend .nav;
+  }
+
+  li {
+    display: inline-block;
```

Comment:
Properties should be ordered background, color, display, margin-right, position, -moz-transition, -ms-transition, -o-transition, -webkit-transition, transition